config/template.rb

```diff
@@ -24,6 +21,39 @@
 apply "config/environments/test.rb"
 template "config/environments/staging.rb.tt"
 
-route 'root "home#index"'
-route %Q(mount Sidekiq::Web => "/sidekiq" # monitoring console\n)
-route "require 'sidekiq/web'"
+route <<-EO_ROUTES
```

Comment:
I think that since this route definition includes three different routes:

1. Lighthouse workaround
2. Sidekiq/web
3. root route

I think we should probably define these with three different route definitions, so that they are inherently three different routes